README.md

```diff
@@ -154,6 +155,24 @@ export const SomeStory = {
 };
 ```
 
+### runOptions
+
+Allows use of any of the available [`axe.run`](https://www.deque.com/axe/core-documentation/api-documentation/#options-parameter) options. See the link for more details. When using `runOptions.rules` in combination with `disabledRules`, **`disabledRules` will always take precedent.**
+
+```jsx
+
+export const SomeStory = {
+  parameters: {
+    axe: {
+      runOptions: {
```

Comment:
Can you also add an entry to https://github.com/chanzuckerberg/axe-storybook-testing/blob/fbb4f02283eda1c59d8564d3de2ba46eed6798d3/CHANGELOG.md ?